teuthology/test/test_misc.py

```diff
@@ -26,45 +27,36 @@ def test_sh_truncate(caplog):
 
 def test_sh_fail(caplog):
     with pytest.raises(subprocess.CalledProcessError) as excinfo:
-        misc.sh("/bin/echo -n AB ; /bin/echo C ; exit 111", 2) == "ABC\n"
+        assert misc.sh("/bin/echo -n AB ; /bin/echo C ; exit 111", 2) == "AB\nC\n"
```

Comment:
Why? 'echo -n' means there should not be a new line after AB

Reply:
Thank you, this is a good catch. I was fixing the lack of asserts in `test_sh_progress`, and made this similar change here. The test passed, but that's because the `assert` never happens since the call raises. I'll revert these two lines